codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/GoWriter.java

```diff
@@ -183,7 +209,7 @@ public GoWriter addDependency(SymbolDependencyContainer dependencies) {
      * @param runnable Runnable that handles actually writing docs with the writer.
      * @return Returns the writer.
      */
-    GoWriter writeDocs(Runnable runnable) {
+    protected GoWriter writeDocs(Runnable runnable) {
```

Comment:
Why is this protected now? The class is final, so it's not like it will ever have inheritors

Reply:
Would private be better? Idea was to prevent this version of the method from being called outside of the `GoWriter`.  In Java the default visibility is package right?

Reply:
Yes, default is package private